airflow/providers/elasticsearch/__init__.py

```diff
@@ -28,7 +28,7 @@
 
 __all__ = ["__version__"]
 
-__version__ = "4.5.1"
+__version__ = "5.0.0"
```

Comment:
Nope. No problem with that change, but yes I remove it too (accidentally committed some elasticsearch "future" changes" - it would have been committed during the future release so no big deal, but would be nice to remove it.